public/app/views/pdf/_digital_object_links.html.erb

```diff
@@ -0,0 +1,12 @@
+<% Array(instances).each do |instance| %>
+    <% next if instance['_inherited'] %>
+    <% if instance['instance_type'] == 'digital_object' && instance['digital_object'] %>
+        <% file_versions = instance['digital_object']['_resolved']['file_versions'] %>
+        <% Array(file_versions).each do |file| %>
+            <% if file['file_uri'] && file['use_statement'] != 'image-thumbnail' %>
```

Comment:
after more thought, it's probably not wise to filter by a use_statement here by default.  it would be better, i think, to filter out any links that are requested to be embedded, unless we were actually embedding images into the PDF, which we said that we weren't going to do for this release.  i'm not sure the exact name of the JSON key here, but we could probably just replace "file['use_statement'] != 'image-thumbnail'" with something like "file['xlink_show_attribute'] != 'embed'" to be more precise.  

either that, or just to include more information about the nature of the link in addition to the file_uri.